examples/v1beta1/taskruns/authenticating-git-commands.yaml

```diff
@@ -201,7 +201,7 @@ spec:
           # When disable-home-env-overwrite is "true", creds-init credentials
           # will be copied to /home/nonroot/.ssh by the entrypoint. We just need to
           # overwrite the known_hosts file with that of our test git server.
-          cp /messages/known_hosts /home/nonroot/ssh/known_hosts
+          cp /messages/known_hosts /home/nonroot/.ssh/known_hosts
```

Comment:
Heh, this case was not tested before, at least not in the examples?
Even though the default has changed, we should still test the case of `disable-home-env-overwrite=false` otherwise we will not now if we inadvertently break it. It's probably still tested via unit tests?

Reply:
hm, this takes us back to the problem of matrix testing behavioural flags.

Reply:
I'll see if this is exercised elsewhere already.

Reply:
I've updated the `pkg/pod/pod_test.go` file to ensure we check that the pod builder does set `"/tekton/home"` as the default HOME var when it's told to.

Beyond that I can't think of a good way to end-to-end test the branches of this specific example without spinning up a new deployment with the flag "true" and another with flag "false".

Reply:
Thank you!